sDownloads.ps1

```diff
@@ -0,0 +1,27 @@
+# Check directory sDownloads.ps1 runs from for sd-config file
+$ConfigPath = $PSScriptRoot + "\sd-config"
+
+# sd-config needs to stay with sDownloads.ps1
+# Change the path if you want sd-config somewhere else
+$Config = Import-Csv -Path $ConfigPath
+
+# Will work fine as long as you use the default Downloads folder
+$Downloads = "C:\Users\$env:UserName\Downloads\"
+
+$Files = $Downloads | Get-ChildItem
+$FileNames = $Files.Name
+
+ForEach($e in $Config) {
```

Comment:
It may or may not be standard, but using a more descriptive name than $e for this variable would be useful. I'm not sure what "$e" is here (I mean, I can infer what it is, but it's not immediately apparent).